action.yml

```diff
@@ -4,7 +4,19 @@ description: 'Authenticate to Azure and run your Az CLI or Az
 inputs: 

```

Comment:
@anatbal creds_resource_manager_endpoint_url should also be added to input fields

Reply:
Thanks @BALAGA-GAYATRI , I fixed it